pkg/monitoring/domainstats/prometheus/prometheus_test.go

```diff
@@ -1187,6 +1187,45 @@ var _ = Describe("Prometheus", func() {
 			Expect(result.Desc().String()).To(ContainSubstring("kubevirt_vmi_filesystem_used_bytes"))
 			Expect(ch).To(BeEmpty())
 		})
-	})
 
+		It("should expose CPU usage metrics", func() {
+			ch := make(chan prometheus.Metric, 3)
+			defer close(ch)
+
+			ps := prometheusScraper{ch: ch}
+
+			domainStats := &stats.DomainStats{
+				Cpu: &stats.DomainStatsCPU{
+					TimeSet:   true,
+					Time:      123000000000,
+					UserSet:   true,
+					User:      456000000000,
+					SystemSet: true,
+					System:    789000000000,
+				},
+				Memory: &stats.DomainStatsMemory{},
+				Net:    []stats.DomainStatsNet{},
+				Vcpu:   []stats.DomainStatsVcpu{},
+			}
+
+			vmi := k6tv1.VirtualMachineInstance{}
+			ps.Report("test", &vmi, newVmStats(domainStats, nil))
+
+			metrics := map[string]float64{
+				"kubevirt_vmi_cpu_usage_seconds":        123,
+				"kubevirt_vmi_cpu_user_usage_seconds":   456,
+				"kubevirt_vmi_cpu_system_usage_seconds": 789,
+			}
+
+			for k, v := range metrics {
```

Comment:
@avlitman @machadovilaca From what I see the unit-tests lane is failing on that suite. I think its a real problem because you cannot expect the metrics that returned from the `ch` to be in the same order at these you defined in `metrics := map[string]float64{` I would suggest to split the test to a test table, in which row you will load a single CPU stat in the domain stats, and expect the same metric to be read from the channel

Reply:
I will work with @machadovilaca on this tomorrow.

Reply:
Fixed with @enp0s3